testing/src/main/java/net/consensys/linea/testing/MultiBlockExecutionEnvironment.java

```diff
@@ -109,7 +109,10 @@ public MultiBlockExecutionEnvironmentBuilder addBlock(
         blockHeaderBuilder.parentBeaconBlockRoot(
             startingBlockNumber$value == 0 ? Bytes32.ZERO : Bytes32.fromHexString("0xBADDADD7"));
       } else {
-        blockHeaderBuilder.parentBeaconBlockRoot(blocks.getLast().header().parentBeaconBlockRoot());
+          // e.g. 0xaaaa..aa00e3 for block number 0xe3
+          final String prefix = "aa".repeat(30);
+          final String suffix = String.format("%04x", blocks.size());
+        blockHeaderBuilder.parentBeaconBlockRoot(Bytes32.fromHexString("0x" + prefix + suffix));
```

Comment:
This makes the beacon root non constant for multi block tests. Also, it imposes all of them to be nonzero save for that of the genesis block.